src/Magnum/OvrIntegration/CMakeLists.txt

```diff
@@ -46,6 +46,10 @@ set(MagnumOvrIntegration_HEADERS
 
     visibility.h)
 
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${CORRADE_CXX_FLAGS}")
+
+include_directories(${MAGNUM_INCLUDE_DIRS} ${MAGNUM_APPLICATION_INCLUDE_DIRS} ${MAGNUM_AUDIO_INCLUDE_DIRS} ${CMAKE_CURRENT_BINARY_DIR})
+
```

Comment:
Ah, I missed those. I was really surprised how this missing made it to code review, should have investigated further, I guess. Will remove it again, sorry ;)